### ISSUE TITLE
Bronchopulmonary anastomosis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6524]
**Preferred term label:** Bronchopulmonary anastomosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Arterial anastomosis (connections) between bronchial and pulmonary arteries PMID: 10598843

**Parent term (use hpo.jax.org/app)**
Abnormality of bronchial artery (to be created) 
Abnormal pulmonary artery morphology HP:0030966

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Chronic Thromboembolic Pulmonary Hypertension ORPHA:70591

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added term
